webapp/graphite/finders/__init__.py

```diff
@@ -28,6 +28,18 @@ def _deduplicate(entries):
       yield entry
 
 
+def extract_variants(pattern):
+  """Extract the pattern variants (ie. {foo,bar}baz = foobaz or barbaz)."""
+  v1, v2 = pattern.find('{'), pattern.find('}')
+  if v1 > -1 and v2 > v1:
+    variations = pattern[v1+1:v2].split(',')
+    variants = [ pattern[:v1] + v + pattern[v2+1:] for v in variations ]
+
+  else:
+    variants = [ pattern ]
```

Comment:
Oh, I didn't see that ``glob`` was use in the finder instead.  I had a ``for fs_path in extract_variants`` pattern in mind.